<launch>

  <!-- defines name of the namespace of the drone -->
  <arg name="UAV_NAME" default="$(optenv UAV_NAME uav1)" />

    <!-- Namespace - all topics, services and parameters will be remapped using this namespace as a prefix (eg.: "/waypoint_flier_simple/odom_uav_in" to "/uav1/waypoint_flier_simple/odom_uav_in") -->
  <group ns="$(arg UAV_NAME)">

    <node pkg="bump_tolerant_control" type="ImuListener" name="imu_listener" output="screen">

    </node>

  </group>

</launch>